products/jsa/features/bootstrap/SWSDrupalContext.php

```diff
@@ -1 +1 @@
-../../../includes/bootstrap/SWSDrupalContext.php
\ No newline at end of file
+/Users/kmakitan/Documents/behat/linky_clicky/includes/bootstrap/SWSDrupalContext.php
```

Comment:
This link won't work on my machine because I'm not running in your account. Please create this link to be relative only to the project.